<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapChooseWindow</class>
 <widget class="QDialog" name="MapChooseWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>当前地图：xxx.json</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_MapChoose">
     <property name="text">
      <string>选择地图（暂时不用）</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_Level1">
     <property name="text">
      <string>关卡1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_Level2">
     <property name="text">
      <string>关卡2</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_Enter">
     <property name="text">
      <string>进入关卡</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_Back">
     <property name="text">
      <string>返回上页</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
